src/ios-deploy/ios-deploy.m

```diff
@@ -192,7 +192,7 @@ void NSLogJSON(NSDictionary* jsonDict) {
             [jsonString writeToFile:@"/dev/stdout" atomically:NO encoding:NSUTF8StringEncoding error:nil];
             [jsonString release];
         } else {
-            [@"{\"error\": \"JSON error\"}" writeToFile:@"/dev/stdout" atomically:NO encoding:NSUTF8StringEncoding error:nil];
+            [@"{\"JSONError\": \"JSON error\"}" writeToFile:@"/dev/stdout" atomically:NO encoding:NSUTF8StringEncoding error:nil];
```

Comment:
Nice fix... That would have been fun to try to catch("error" instead of "Error" where if you were probably also expecting a "Code"...)